cmd/segment-reaper/bitmask_test.go

```diff
@@ -260,5 +260,47 @@ func TestBitmask(t *testing.T) {
 			ok := mask.IsSequence()
 			assert.False(t, ok)
 		})
+
+	})
+	t.Run("Unset", func(t *testing.T) {
+		t.Run("ok", func(t *testing.T) {
+			var (
+				expectedIdx = rand.Intn(64)
+				mask        bitmask
+			)
+
+			err := mask.Set(expectedIdx)
```

Comment:
Let's set another random index and verify after `Unset` call that the index remains.

Reply:
done